src/main/java/uk/gov/hmcts/reform/cwrdapi/util/RequestUtils.java

```diff
@@ -30,4 +41,35 @@ public static void trimIdamRoles(List<CaseWorkersProfileCreationRequest> request
             }
         });
     }
+
+    public static PageRequest validateAndBuildPaginationObject(Integer pageNumber,
+                                                               Integer pageSize,
+                                                               String sortColumn,
+                                                               String sortDirection,
+                                                               String loggingComponentName,
+                                                               int configPageSize,
+                                                               String configSortColumn) {
+
+        if (pageNumber != null && pageNumber < 0) {
+            log.info("{}:: Invalid Page Number", loggingComponentName);
```

Comment:
common method in util class can be written to avoid duplication of code for checking page side and number

Reply:
The whole method `validateAndBuildPaginationObject` itself is in util class, so that validation and building pagination object can be done for any input param sent